strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/HttpConnectionPoolConfigurationManagementController.java

```diff
@@ -50,7 +50,7 @@
                                          message = "Repository doesn't have remote repository!"),
                             @ApiResponse(code = 404,
                                     message = "The (storage/repository) does not exist!") })
-    @PutMapping(value = "{storageId}/{repositoryId}/{numberOfConnections}",
+    @PutMapping(value = "/{storageId}/{repositoryId}/{numberOfConnections}",
```

Comment:
Any particular reason you've changed these paths to start with a `/`?

Reply:
I've just added them to keep coherence with the rest of the controllers, who also have it.

Reply:
Actually, the rest of the controllers (outside this pull) are all defining them relatively without a starting slash, which is why I asked. Would you mind changing this back for consistency?